deploy/charts/external-secrets/templates/rbac.yaml

```diff
@@ -35,6 +36,7 @@ rules:
     - ""
```

Comment:
We need additional RBAC rules to be able to patch the status:

```diff
diff --git a/deploy/charts/external-secrets/templates/rbac.yaml b/deploy/charts/external-secrets/templates/rbac.yaml
index 34394da5..9a22df9d 100644
--- a/deploy/charts/external-secrets/templates/rbac.yaml
+++ b/deploy/charts/external-secrets/templates/rbac.yaml
@@ -29,6 +29,9 @@ rules:
     - "clustersecretstores"
     - "clustersecretstores/status"
     - "clustersecretstores/finalizers"
+    - "clusterexternalsecrets"
+    - "clusterexternalsecrets/status"
+    - "clusterexternalsecrets/finalizers"
     verbs:
     - "update"
     - "patch"
```

Reply:
Added these to the RBAC.